### ISSUE TITLE
Split "Documents" in dropdown into "Forms" and "Reports"

### ISSUE THREAD

**Issue by mattslay** [DougHennig/ProjectExplorer#7]
I'd also like to suggest that you split the "documents" scope into 2 *separate* entries. Replace "Documents" with "Forms" and "Reports".  Just because native PM groups those 2 into one does not mean the PE needs to.  I often find that when I am on Documents, then I have Forms expanded in the treeview, and I want to get to Reports, I have to scroll way down below all the Forms just to get to Reports. To me, grouping Forms and Reports together is just not a great grouping.

![2017-05-24_17-31-22](https://cloud.githubusercontent.com/assets/309016/26428475/8371d94c-40a7-11e7-8b8c-88a4e8fac342.png)
g.

**Comment by DougHennig** [comment]
You can do that yourself: open the Tag Editor in the File menu, create Forms and Reports tags, then remove Documents from the tags for your forms and add Forms instead. You'll still see Documents in the tags list but there won't be anything in that tag once you've retagged your forms.

**Comment by mattslay** [comment]
Now *that* is a great feature.  Tags !!! Nice.

**Comment by mattslay** [comment]
I've added the tags, and I made a few manual updates through the UI. However, I'd like to hack one of the tables somewhere to do mass-updates to change the Tag for all SCX files to "Forms" and all FRX files to "Reports".

I see a dbf named "Blah_MetaData.dbf" and I see a Key column there and the Tags memo field. So, where I can resolve what filename the Key points to? With that info, I can do the hack that I want to do.
